setup.py

```diff
@@ -3,66 +3,46 @@
 import os
 import sys
 
-import jwt
-
-from setuptools import setup
-from setuptools.command.test import test
-
-
-class PyTest(test):
-    user_options = [('pytest-args=', 'a', "Arguments to pass to py.test")]
-
-    def initialize_options(self):
-        test.initialize_options(self)
-        self.pytest_args = []
-
-    def finalize_options(self):
-        test.finalize_options(self)
-        self.test_args = []
-        self.test_suite = True
-
-    def run_tests(self):
-        # import here, cause outside the eggs aren't loaded
-        import pytest
-        errno = pytest.main(['.'] + self.pytest_args)
-        sys.exit(errno)
+from setuptools import find_packages, setup
 
+__version__ = None
```

Comment:
Let's change this for what I proposed earlier. [`get_version()`](https://github.com/tomchristie/django-rest-framework/blob/master/setup.py#L11) works well.